cli/main.c

```diff
@@ -919,10 +919,32 @@ static int arbitration_set(int argc, char **argv)
 	return 0;
 }
 
+static void print_bind_info(struct switchtec_bind_status_out status)
+{
+	int result = status.bind_state & 0x0F;
+	int state = (status.bind_state & 0xF0) >> 4;
+
+	switch (result) {
+	case 0:
```

Comment:
We should probably have an enum for these case values somewhere.

Reply:
I agree it. Thanks